wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -127,13 +127,6 @@
    */
   public static final String SIDECAR_MEMORY_LIMIT_ATTR_TEMPLATE = "sidecar.%s.memory_limit";
 
-  /**
-   * Template for workspace config attribute key that stores plugin component preference. The first
-   * %s should be replaced with `pluginPublisher/pluginName` and the second one should be replaced
-   * with preference name. Example value: `plugin.redhat/java.preference.java.home`.
-   */
-  public static final String PLUGIN_PREFERENCE_ATTR_TEMPLATE = "plugin.%s.preference.%s";
```

Comment:
Why do we remove this? Isn't `che-theia` still reading workspace config instead of devfile?